Makefile

```diff
@@ -85,4 +85,7 @@ website: ## Generate Docs Website
 toolchain: ## Install developer toolchain
 	./hack/toolchain.sh
 
+check-helm-docs: ## Check if chart README is staled
+	./hack/check-helm-docs.sh
```

Comment:
One final nitpick. Can you just inline this in `make codegen` to avoid an unnecessary target?